### PR TITLE
Add a survey request to top of search results

### PR DESCRIPTION
**Pull request by robmckinnon**
Display request to do separating parents survey on the search results page for users who have chosen 'children', 'divorce', 'domestic abuse' as the area of law.

### CODE REVIEW

File: courtfinder/apps/search/templates/search/results.jinja [hunk 1]
```diff
@@ -21,11 +21,20 @@
 </div>
 <div id="search-results-page" class="content inner cf">
   <header class="page-header">
-    <h1>Search results</h1>
+    {# --- temporary survey request > #}
+    {% if aol == 'Children' or aol == 'Domestic violence' or aol == 'Divorce' %}
+      <p class="info-box" style="float:right; display:inline; width: 40%; padding: 20px 15px 20px 80px; margin-left: 50px; background: #EDEDED url('/assets/images/icon-information-2x.png') no-repeat scroll 15px 20px;">
+        Are you a separating parent?<br>
```
Comment (maxf) [inline courtfinder/apps/search/templates/search/results.jinja]: Does that render well in IE8?

Comment (robmckinnon) [inline courtfinder/apps/search/templates/search/results.jinja]: Analytics show that 2.7% of user sessions are IE8.

No explicit test has been done in IE8. The styles are taken from a similar info box on people finder. This is intended to be a temporary survey link that will be removed later.

**Comment by robmckinnon** [comment]
It is not clear to me from the shippable build output what has failed: https://app.shippable.com/builds/56b35bcae72c740d007166c6

File: courtfinder/apps/search/templates/search/results.jinja (hunk 1 above)
Comment (maxf) [inline courtfinder/apps/search/templates/search/results.jinja]: Fair enough. Keep in mind that many courtfinder users are court/moj staff who use IE7.

**Pull request merged**